Library/brew.sh

```diff
@@ -167,7 +167,7 @@ then
   # Hide shellcheck complaint:
   # shellcheck source=/dev/null
   source "$HOMEBREW_BASH_COMMAND"
-  { "homebrew-$HOMEBREW_COMMAND" "$@"; exit $?; }
+  { "homebrew_$HOMEBREW_COMMAND" "$@"; exit $?; }
```

Comment:
Should we also escape `HOMEBREW_COMMAND`? There is no need for now but may help in the future.